src/main/kotlin/org/dashj/platform/dpp/errors/concensus/ConcensusException.kt

```diff
@@ -0,0 +1,150 @@
+//                Codes.InvalidAssetLockProofTransactionHeightError -> InvalidAssetLockProofTransactionHeightException(arguments)
+//
+//                Codes.InvalidIdentityAssetLockTransactionError -> InvalidIdentityAssetLockTransactionException(arguments)
+//                Codes.InvalidIdentityAssetLockTransactionOutputError -> InvalidIdentityAssetLockTransactionOutputException(arguments)
+//                Codes.InvalidIdentityPublicKeyDataError -> InvalidIdentityPublicKeyDataException(arguments)
+//                Codes.InvalidInstantAssetLockProofError -> InvalidInstantAssetLockProofException(arguments)
+                Codes.InvalidInstantAssetLockProofSignatureError ->
+                    InvalidInstantAssetLockProofSignatureException(arguments)
+//                Codes.InvalidStateTransitionTypeError -> InvalidStateTransitionTypeException(arguments)
+//                Codes.MissingStateTransitionTypeError -> MissingStateTransitionTypeException(arguments)
+//                Codes.StateTransitionMaxSizeExceededError -> StateTransitionMaxSizeExceededException(arguments)
+//                Codes.IdentityNotFoundError -> IdentityNotFoundException(arguments)
+//                Codes.InvalidIdentityPublicKeyTypeError -> InvalidIdentityPublicKeyTypeException(arguments)
+                Codes.InvalidStateTransitionSignatureError -> InvalidStateTransitionSignatureException()
+//                Codes.MissingPublicKeyError -> MissingPublicKeyException(arguments)
+                Codes.BalanceIsNotEnoughError -> BalanceIsNotEnoughException(arguments)
+//                Codes.DataContractAlreadyPresentError -> DataContractAlreadyPresentException(arguments) */
+//                Codes.DataTriggerConditionError -> DataTriggerConditionException(arguments)
+//                Codes.DataTriggerExecutionError -> DataTriggerExecutionException(arguments)
+//                Codes.DataTriggerInvalidResultError -> DataTriggerInvalidResultException(arguments)
+                Codes.DocumentAlreadyPresentError -> DocumentAlreadyPresentException(arguments)
+//                Codes.DocumentNotFoundError -> DocumentNotFoundException(arguments)
+//                Codes.DocumentOwnerIdMismatchError -> DocumentOwnerIdMismatchException(arguments)
+//                Codes.DocumentTimestampsMismatchError -> DocumentTimestampsMismatchException(arguments)
+//                Codes.DocumentTimestampWindowViolationError -> DocumentTimestampWindowViolationException(arguments)
+//                Codes.DuplicateUniqueIndexError -> DuplicateUniqueIndexException(arguments)
+                Codes.InvalidDocumentRevisionError -> InvalidDocumentRevisionException(arguments)
+                Codes.IdentityAlreadyExistsError -> IdentityAlreadyExistsException(arguments)
+//                Codes.IdentityPublicKeyAlreadyExistsError -> IdentityPublicKeyAlreadyExistsException(arguments)
+                else -> throw UnknownConcensusError(code.code, arguments)
```

Comment:
All those commented errors, are they gonna be used in the future?

Reply:
Some of them will.  Currently, the ones that are not used by any other layer of our system are commented out.